db/controller.js

```diff
@@ -21,4 +21,20 @@ const getAllEndpoints = (req, res, next) => {
     })
 }
 
-module.exports = {getAllTopics, getAllEndpoints}
\ No newline at end of file
+const getArticleById = (req, res, next) => {
+    const { article_id } = req.params;
+
+    if ( isNaN(article_id)) {
```

Comment:
This is unnecessary, isNan is a little problematic and shouldn't really be used. I can totallly see why you want to use it but it is unnecessary.

In this case if article_id is not a number then PSWL will throw an error anyway